demos/selenium-test-app/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "selenium-test-app",
+  "name": "selenium_test_app",
   "private": true,
   "dependencies": {
-    "@dfinity/agent": "^0.11.3",
-    "@dfinity/auth-client": "^0.11.3",
-    "@dfinity/authentication": "^0.11.3",
-    "@dfinity/candid": "^0.11.3",
-    "@dfinity/identity": "^0.11.3",
-    "@dfinity/principal": "^0.11.3",
+    "@dfinity/agent": "^0.12.1-beta.0",
```

Comment:
This is required because only the beta version supports `derivationOrigin`. This is required to test the fast-tracked feature https://dfinity.atlassian.net/browse/L2-712

Reply:
👍